src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -101,6 +101,21 @@ def rename(self, name: str, new_name: str):
                 component='cephfs')
         return f'Volume {name} renamed successfully to {new_name}'
 
+    @UpdatePermission
+    @Endpoint('PUT')
+    def auth(self, name: str, client_id: int, caps: List[str], root_squash: bool):
+        if root_squash:
+            caps.insert(2, 'root_squash')
+        error_code, _, err = mgr.mon_command({'prefix': 'fs authorize',
+                       'filesystem': name,
+                       'entity': f'client.{client_id}',
```

Comment:
>its actually not good to prefix client without informing the user. 

Yeah, I get that, that's why I was hesitant on adding this and how to approach it.

But actually, when talking about cephx this is usually referred as `USERTYPE.USERID` ,  `TYPE.ID` or similar.
That's why I left is as userId since that's what I'm actually asking for in the form as well as in the API.

>Or check if the input starts with client. in UI, if it is send it. else add that as well. You can evaluate what's best based on your experience with this
I'm not a fan of having differnt inputs return same output, I think it can be confusing or missleading.

Anyways, @nizamial09 , how would you prefer to do it? I'll just do it as you tell me

Reply:
> But actually, when talking about cephx this is usually referred as USERTYPE.USERID , TYPE.ID or similar.

we can't really expect most people to know that either. For them, they can do whatever from the UI and they can expect it to work correctly (And UI is responsible for all those validations). 

> I'm not a fan of having differnt inputs return same output, I think it can be confusing or missleading.

Understood! Since client id and client type are two different things, why not have a disabled input that says `Client Type` and show `client`? Probably that is simple and less confusing for user. wdyt? @Pegonzal

Reply:
I was also thinking about that approach, I guess is the most cleare one, so I'll make the changes for it. Thanks Nizam